color-spantrace/Cargo.toml

```
@@ -3,6 +3,7 @@ name = "color-spantrace"
 version = "0.2.2"
 description = "A pretty printer for tracing_error::SpanTrace based on color-backtrace"
 documentation = "https://docs.rs/color-spantrace"
+include = ["READEME.md", "src/**/*.rs", "build.rs"]
```

```suggestion
include = ["README.md", "src/**/*.rs", "build.rs"]
```

Do we need to add licenses in this?

---

Yes, this should include the licenses. I've pushed a fixed version